src/eslint-plugin-adeira/README.md

```diff
@@ -106,3 +106,17 @@ Example of **correct** code:
 // given the file Kiwi_data.graphql.js exports type Kiwi_data
 import type { Kiwi_data } from './__generated__/Kiwi_data.graphql';
 ```
+
+## valid-test-folder
+
+This rule checks that your test files are in a folder called __tests__
+
+Example of **incorrect** filename:
+
+- `/src/__test__/my.test.js`
```

Comment:
What if I set `testMatch` jest option to `'<rootDir>/**/__test__/**/*.js?(x)',`?

Reply:
Yeah, there is probably room for using some config here

Reply:
oki